src/api/routes/NetworkController.ts

```diff
@@ -0,0 +1,41 @@
+import { Connection } from '@solana/web3.js';
+import { Router, Request, Response, NextFunction} from 'express';
+import { Logger } from 'winston';
+import { Container} from 'typedi';
+import NetworkService from '../../service/NetworkService';
+import { NetworkIdentifier } from '../../interfaces/NetworkIdentifier';
+
+const route = Router();
+
+export default (app: Router) => {
+  app.use('/network', route);
+
+
+  route.post('/list', (req: Request, res: Response) => {
```

Comment:
Might wanna change this to `route.get` based on https://www.rosetta-api.org/docs/NetworkApi.html ?